src/components/StokesAnalysis/StokesAnalysisComponent.tsx

```diff
@@ -512,32 +526,25 @@ export class StokesAnalysisComponent extends React.Component<WidgetProps> {
         return a === b && a === c && a === d && a !== null;
     }
 
-    private getScatterColor(percentage: number, frequencyIncreases: boolean): string {
+    private getScatterColor(percentage: number, redToBlue: boolean): string {
         const colorMap = this.widgetStore.colorPixel.color;
         const mapSize = this.widgetStore.colorPixel.size;
-        if (!frequencyIncreases) {
+        if (redToBlue) {
```

Comment:
I don't think redToBlue is a good name here, because it depends on the colormap itself. Rather just say "reversed"